.gitignore

```diff
@@ -2,3 +2,8 @@ node_modules/
 svg/output/
 .idea/
 npm-debug.log
+svg-min/
+icon-font/
+build/
+svg-sprite/
+react/social-logo/index.jsx
```

Comment:
Do we even need to generate this file anymore? Just seems strange with its placement adjacent the source (pre-build) files.